tests/resume.test.ts

```diff
@@ -220,3 +220,63 @@ describe('patch resume', () => {
     ).rejects.toThrowError(ForbiddenError);
   });
 });
+
+describe('delete resume', () => {
+  test('delete resume route successfully deletes user resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    const resume = ResumeFactory.fake({ isResumeVisible: false, user: member });
+    await new PortalState()
+      .createUsers(member)
+      .createResumes(member, resume)
+      .write();
+
+    const params = { uuid: resume.uuid };
+
+    const storageService = Mocks.storage();
+    const resumeController = ControllerFactory.resume(
+      conn,
+      instance(storageService),
+    );
+    await resumeController.deleteResume(params, member);
+
+    const repository = conn.getRepository(ResumeModel);
+    const resumesStored = await repository.find({ user: member });
+
```

Comment:
nit: instead of using the `ResumeRepository`, let's stick with our convention of using `conn.manager.findOne` as we do in other tests (e.g. [here](https://github.com/acmucsd/membership-portal/blob/master/tests/merchOrder.test.ts#L194-L199))

Reply:
Sounds good, I think I should use find to be consistent with future for scalable amount of resumes.